force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js

```diff
@@ -328,7 +328,7 @@ export default class RecurringDonationTable extends LightningElement {
         });
 
         this.isElevateDonation = this.currentRecord.recurringDonation.CommitmentId__c ? true : false;
-        this.isInitiallyMonthlyDonation = this.currentRecord.recurringDonation.Day_of_Month__c ? true : false;
+        this.isInitiallyMonthlyDonation = this.currentRecord.recurringDonation.npe03__Installment_Period__c === 'Monthly';
```

Comment:
Please use a constant for "Monthly" like we do in the other classes

Reply:
Ok, I updated it.